### PR TITLE
✨🐛UpScore Analytics: Changed ampURL to canonical URL and added 2 custom params

### PR DESCRIPTION
**Pull request by suransandi**
Changed {ampURL} to {canonicalURL} and added 2 custom params. 
Adding video tracking events.

### CODE REVIEW

**Comment by suransandi** [comment]
Changed {ampURL} to {canonicalURL} and added 2 custom params

**Comment by suransandi** [comment]
Hi @lannka . How are you? 
Is there something that must be changed from my side to resolve the remaining issue?

**Comment by suransandi** [comment]
Hi @zhouyx, 
How are you? Is there any chance for us to have this pull request merged?
Please let me know if there is anything that needs to be done from my side to resolve the blocking issue. 
Thank you.

**Comment by zhouyx** [comment]
@suransandi Sorry for the delay here and thank you for the ping!  Added @micajuine-ho for review.

File: examples/analytics-vendors.amp.html [hunk 1]
```diff
@@ -1522,13 +1522,41 @@
...
+          "video_name" : "test test test",
+          "video_thumbnail" : "poster_test"
+        }
+      },
+      "videoPauses": {
+        "on": "video-pause",
+        "selector": "#myVideo",
+        "request": "video-pause"
+      },
+
```
Comment (micajuine-ho) [inline examples/analytics-vendors.amp.html]: nit: remove extra line

File: examples/analytics-vendors.amp.html [hunk 2]
```diff
@@ -1522,13 +1522,41 @@
...
+  
+      "videoSecondsPlayed": {
+        "on": "video-seconds-played",
+        "selector": "#myVideo",
+        "videoSpec": {
+          "exclude-autoplay": false,
+          "interval": 10
+        },
+        "request": "video-info"
+
```
Comment (micajuine-ho) [inline examples/analytics-vendors.amp.html]: nit: remove extra line

File: extensions/amp-analytics/0.1/test/vendor-requests.json [hunk 3]
```diff
@@ -437,13 +437,20 @@
...
+      "videoBase":"https://hit-pool.upscore.com/amp?url=_canonical_url_&video_id=!video_id",
+      "videoTime":"ct=!currentTime&d=!duration",
+      "video-play":"https://hit-pool.upscore.com/amp?url=_canonical_url_&video_id=!video_id&e=video_start&ct=!currentTime&d=!duration&video_name=!video_name&video_thumb=!video_thumbnail",
+      "video-pause":"https://hit-pool.upscore.com/amp?url=_canonical_url_&video_id=!video_id&e=video_pause&ct=!currentTime&d=!duration",
+      "video-stop":"https://hit-pool.upscore.com/amp?url=_canonical_url_&video_id=!video_id&e=video_stop&ct=!currentTime&d=!duration",
+      "video-info":"https://hit-pool.upscore.com/amp?url=_canonical_url_&video_id=!video_id&e=video_position&ct=!currentTime&d=!duration"
+
```
Comment (micajuine-ho) [inline extensions/amp-analytics/0.1/test/vendor-requests.json]: nit: remove extra line

**Comment by suransandi** [comment]
Hi @micajuine-ho thank you for approving the changes. 
Also I apologize for my rather late update. 
Almost month ago this was a top priority task to finish on my list, but was pushed aside by other work.
When can I expect this changes to be in production release?

**Comment by lannka** [comment]
@suransandi FYI this will go to production in 1-2 weeks.

**Pull request merged**